nautilus_core/persistence/src/bin/init_db.rs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+//  Copyright (C) 2015-2023 Nautech Systems Pty Ltd. All rights reserved.
+//  https://nautechsystems.io
+//
+//  Licensed under the GNU Lesser General Public License Version 3.0 (the "License");
+//  You may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at https://www.gnu.org/licenses/lgpl-3.0.en.html
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------------------------------
+
+use anyhow::Result;
+use dotenv::dotenv;
+use nautilus_persistence::db::database::{init_db_schema, Database};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // load envs if exists
```

Comment:
> added two new actions to Makefile and that are init-db and drop-db that will initialize Nautilus SQL schema to connected database ( mostly Postgres here). We will need those actions to resync the latest schema with the database. Migrations of schema will not be currently supported

You can consider using sqlx built in migrate functionality which applies migrations to a db on startup. https://docs.rs/sqlx/latest/sqlx/macro.migrate.html

Reply:
Yeah, good point. I know about this, but migration functionality will come at the later stages when we finish building Nautilus SQL data model, and when schema becomes more stable and less changing. And when there will actual db state outside nautilus which needs to be preserved but migrated to the latest schema.

Reply:
Thanks for the heads up @twitu, this will be very useful at some point.